.pre-commit-config.yaml

```diff
@@ -1,15 +1,20 @@
 repos:
 - repo: https://github.com/ambv/black
-  rev: stable
+  rev: 19.10b0
   hooks:
   - id: black
     language_version: python3.7
 - repo: https://github.com/timothycrosley/isort
-  rev: 4.3.21-2
+  rev: 4.3.21
```

Comment:
I can't recall what the different was between `.21` and `.21-2`, I think the latter was a fixed github hook, while they both use unsuffixed pypi version...